app/src/main/java/com/eis/audioUtility/AudioUtilityManager.java

```diff
@@ -0,0 +1,199 @@
+package com.eis.audioUtility;
+
+/**
+ * This is an Utility class used to manage main Audio Streams (Alarm, Music, and Ring),
+ * and to manage Vibration
+ *
+ * @author Francesco Bau'
+ */
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+import static android.content.Context.AUDIO_SERVICE;
+
+
```

Comment:
What about a class description?

Reply:
Oh wait it's up there, you should move it attached to the class name, on this line

Reply:
Ok will do it. Thank you.